doc/zstd_compression_format.md

```diff
@@ -1440,7 +1440,7 @@ __`Entropy_Tables`__ : follow the same format as tables in [compressed blocks].
               Repeat distribution mode for sequence decoding.
               It's finally followed by 3 offset values, populating recent offsets (instead of using `{1,4,8}`),
               stored in order, 4-bytes __little-endian__ each, for a total of 12 bytes.
-              Each recent offset must have a value < dictionary size.
+              Each recent offset must have a value <= dictionary size, and cannot equal 0.
```

Comment:
Since we are into this level of detail,
the limit is actually the dictionary _content_ size.